Week-4/session-10/callBindApply.js

```diff
@@ -0,0 +1,38 @@
+class Calculator{
+  add(num1,num2){
+      return num1+num2
+  }
+  subtract(num1,num2){
+      return num1-num2
+  }
+  multiply(num1,num2){
+      return num1*num2
+  }
+  divide(num1,num2){
+      return num1/num2
+  }
+}
+
+class ScientificCalculator extends Calculator {
+  square(num){
+      return num**2
+  }
+  cube(num){
+      return num**3
+  }
+  power(num,raise){
+      return num**raise
+  }
+  multiplyByTwo(num) {
+      return num * 2;
+  }
+  powerOfThree(num){
```

Comment:
- The ScientificCalculator class contains two methods that do the same thing: cube(num) and powerOfThree(num). It would be better to keep only one of these methods to avoid redundancy and potential confusion.